deploy/Dockerfile

```diff
@@ -1,26 +1,27 @@
 FROM node:16-alpine AS BASE
 WORKDIR /usr/src/app
-
-COPY ../package.json ./
-COPY ../package-lock.json ./
+COPY package.json package-lock.json ./
```

Comment:
The existence of Dockerfile in this directory is at least very unusual here. 

To me it make sense to have such Dockerfile in root project of this folder. On push to master we should trigger this `docker build` which should deploy image to the docker github registry.  Docker-compose within `./deploy` folder should use the image from docker registry directly.

The setting described above you can use without docker registry as follows:
```
cd $project_root
docker build .
cd ./deploy
docker-compose up
```